tox.ini

```
@@ -70,3 +71,10 @@ deps =
     {[testenv]deps}
     dbt-core~=1.6.0
     dbt-postgres~=1.6.0
+
+[testenv:dbt17]
+description = run with dbt==1.7
+deps =
```

Can we set the environment variable here?

```suggestion
deps =
setenv =
    DBT_CLEAN_PROJECT_FILES_ONLY=false
```

---

Done! 10e59ebd21c541c145056a78be21416f63aecd40